sklearn/gaussian_process/kernels.py

```diff
@@ -1152,8 +1152,11 @@ class RBF(StationaryKernelMixin, NormalizedKernelMixin, Kernel):
         used. If an array, an anisotropic kernel is used where each dimension
         of l defines the length-scale of the respective feature dimension.
 
-    length_scale_bounds : pair of floats >= 0, default: (1e-5, 1e5)
-        The lower and upper bound on length_scale
+    length_scale_bounds : pair of floats >= 0 or the string "fixed",
+        default: (1e-5, 1e5)
```

Comment:
This won't appear as part of the type specification, but as part of the description. See http://scikit-learn.org/circle?8956/_changed.html.

Instead do "The lower and upper bound on length_scale (default: (1e-5, 1e5))."